src/traits.d

```diff
@@ -522,6 +546,15 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
     {
         return isFuncX(f => !f.needThis() && !f.isNested());
     }
+    if (e.ident == Id.isModule)
+    {
+        //TODO: can we handle the case when isPkgMod == PKGunknown?
```

Comment:
> we can't tell yet whether it's a package or not, right?

Right. While module load process, `import a.b;` will check:

1. find directory `a` from import paths. If it found,
2. find D source file `b.d` from the directory.

Note that it's not tested the existence of `a/package.d`. Until the name `a` is actually imported as a module (e.g. `import a;`), it's unknown whether it's `package.d` module or not.

You can see the code to resolve `PKGunknown` in `Import.load()` or `Module.parse()`.